bentoml/yatai/db.py

```diff
@@ -43,6 +50,10 @@ def init_db(db_url):
     if is_sqlite_db(db_url):
         extra_db_args['connect_args'] = {'check_same_thread': False}
         extra_db_args['echo'] = False
+    elif is_postgresql_db(db_url):
+        extra_db_args['connect_args'] = {'application_name': 'yatai'}
```

Comment:
Thanks for adding `application_name` to make it easy to read in pg_stats_activity

Reply:
@yubozhao Yes, you can also monitor the connection with
```
select *
from pg_stat_activity;
```